recipes/jupyterlab-classic/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jupyterlab-classic" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "94e83b6fbfcf0e3ad195883ea8794a7ac2135ca7177d6a9e0ebd60ddc21a175a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - jupyter-packaging >=0.7.9
+
+  run:
+    - python >=3.6
+    - jupyterlab >=3.0.0,<4
+
+test:
+  imports:
+    - jupyterlab_classic
+  commands:
+    - pip check
+    - jupyter classic --help
```

Comment:
Perhaps add a `jupyter labextension list` as well, just so there's some record of `data_files` having been moved around properly.

Reply:
Thanks for the suggestion, sounds good :+1: